sycl/doc/extensions/supported/sycl_ext_intel_usm_address_spaces.asciidoc

```diff
@@ -88,6 +88,10 @@ supports.
 `sycl::ext::intel::decorated_host_ptr`, `sycl::ext::intel::device_ptr`,
 `sycl::ext::intel::raw_device_ptr` and `sycl::ext::intel::decorated_device_ptr`.
 `sycl::host_ptr` and `sycl::device_ptr` are deprecated.
+
+|3
+`sycl::host_ptr` and `sycl::device_ptr` are removed.
```

Comment:
```suggestion
|`sycl::host_ptr` and `sycl::device_ptr` are removed.
```

This row does not appear in the HTML render.  I think it is because you are missing the pipe character (`|`) here.